atomicfu-common/src/main/kotlin/kotlinx/atomicfu/AtomicFU.common.kt

```diff
@@ -217,6 +221,8 @@ public expect class AtomicInt {
      */
     public var value: Int
 
+    public val trace: Trace?
```

Comment:
can it be non-nullable to avoid `?.` everywhere?
Default no-op implementation should be fine

Reply:
The trace field shall not be exposed at all. If you need to trace, you explicitly create your own trace instance with `val trace = Trace()` in your class.